instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/__init__.py

```diff
@@ -212,9 +212,16 @@ def _traced_execute_pipeline(func, instance, args, kwargs):
                 span.set_attribute(
                     "db.redis.pipeline_length", len(command_stack)
                 )
-            response = func(*args, **kwargs)
-            if callable(response_hook):
-                response_hook(span, instance, response)
+
+            response = None
+            try:
+                response = func(*args, **kwargs)
+            except redis.WatchError:
+                span.set_status(StatusCode.UNSET)
+            finally:
```

Comment:
You can avoid the finally and save one level of indentation right?

Reply:
yes, updated, thanks for pointing it out